src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+
+  transform: scale(1);
+  transition: transform 300ms ease-in-out;
```

Comment:
According to the checklist, if you use 'transform' or 'transition' properties, make sure they are necessary for the task. Verify if scaling the card is a requirement of the task. If not, consider removing these properties to comply with the checklist.